<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Progress</class>
 <widget class="QWidget" name="Tab_Progress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QTableWidget" name="table_list">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Gen</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mol</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Job ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time Elapsed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enthalpy (eV)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ancestry</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="push_refresh">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Refresh</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_36">
     <property name="text">
      <string>Refresh &amp;period:</string>
     </property>
     <property name="buddy">
      <cstring>spin_period</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spin_period">
     <property name="suffix">
      <string> seconds</string>
     </property>
     <property name="maximum">
      <number>3600</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="push_refreshAll">
     <property name="text">
      <string>Refresh &amp;all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>table_list</tabstop>
  <tabstop>push_refresh</tabstop>
  <tabstop>spin_period</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
